rakefile

```diff
@@ -143,6 +214,8 @@ desc 'Run Performance setup and a very short iteration of gatling tests'
 rototiller_task :acceptance do |t|
   ENV['ENVIRONMENT_TYPE'] = 'gatling'
   ENV['PUPPET_GATLING_SCENARIO'] = 'acceptance.json'
+  ENV['BEAKER_HOSTS'] = 'config/beaker_hosts/pe-vmpooler.cfg'
+  ENV['ABS_OS'] = 'centos-7-x86_64'
```

Comment:
on line 55 you set this if not set... why does it need setting here?  And if here, why on 55?  And why different values.  It may all be good, and if so, some more comments would really help.